src/relations.jl

```diff
@@ -66,19 +66,25 @@ export q_vap_from_RH_liquid
 
 """
     gas_constant_air(param_set, [q::PhasePartition])
+    gas_constant_air(param_set, q_tot, q_liq, q_ice)
 
 The specific gas constant of moist air given
  - `param_set` an `AbstractParameterSet`, see the [`Thermodynamics`](@ref) for more details
  - `q` [`PhasePartition`](@ref). Without this argument, the results are for dry air.
+ - `q_tot`, `q_liq` `q_ice` - specific liquid water contents for total water, liquid water and ice
 """
+@inline function gas_constant_air(param_set::APS, q_tot, q_liq, q_ice)
+    R_d = TP.R_d(param_set)
+    molmass_ratio = TP.molmass_ratio(param_set)
+    return R_d *
+           (1 + (molmass_ratio - 1) * q_tot - molmass_ratio * (q_liq + q_ice))
```

Comment:
This is correct. However, it would be clearer to write it as follows, I think:
```suggestion
    R_v = TP.R_v(param_set)
    q_v = q_tot - q_liq - q_ice
    return R_d * (1 - q_t) * R_v * q_v
```

Reply:
Yes! I didn't want to change the formulation that was already coded in Thermodynamics. But this version is MUCH easier to read.